packages/cannon-worker-api/package.json

```diff
@@ -28,8 +29,9 @@
     "prettier": "prettier --list-different .",
     "prettier-fix": "prettier --write ."
   },
-  "dependencies": {
-    "three": "^0.137.0"
+  "peerDependencies": {
+    "three": "^0.137.0",
+    "cannon-es": "latest"
```

Comment:
This might be important, should three be a dep or peer dep for this?
I also added the explicit cannon-es peer dependency here, might instead be a straight up dependency, I need eyes on this

Reply:
cannon-es is a devDependency. It is bundled into web worker.